docs/interoperability.md

```diff
@@ -64,6 +68,102 @@ const App = kind({
 });
 ```
 
+You can even use `Moonstone` themed components after installing `@enact/moonstone`.
+
+```javascript
+import BodyText from '@enact/moonstone/BodyText';
+import Button from '@enact/moonstone/Button';
+import { MoonstoneDecorator } from '@enact/moonstone/MoonstoneDecorator';
+import React, { Component } from 'react';
+
+class App extends Component {
+	render() {
+		return (
+			<div>
+				<BodyText centered>
+					These are Enact Moonstone components in a CRA app
+				</BodyText>
+				<Button>Click me</Button>
+			</div>
+		);
+	};
+}
+
+export default MoonstoneDecorator(App);
+```
+
+### Styling Enact Components Outside of Enact
+
+Behind the scenes, Enact supports using CSS modules and most Enact components provide public class
+names that can be overridden to easily change their style.
+
+#### CRA Example
+
+If you are using `react-scripts` version `2.0.0` or later, you can import style overrides by using
+the `[component].module.css` filename convention.  For earlier versions, you will need to eject
+the CRA app and modify the webpack configuration to enable CSS modules.  This [article](https://medium.com/nulogy/how-to-use-css-modules-with-create-react-app-9e44bec2b5c2)
+is a good reference for enabling CSS modules prior to `2.0.0`. 
```

Comment:
Should this be `2.3.0`?

Reply:
It's about the version of `react-scripts`.  Apps made with `create-react-app` that used `react-scripts` prior to 2.0.0 couldn't use this feature.

Reply:
:disappear: